src/smart-components/request/requests.js

```diff
@@ -29,13 +29,13 @@ const Requests = ({ fetchRequests, requests, pagination, history }) => {
   const routes = () => <Fragment>
     <Route exact path="/requests/add_comment/:id" render={ props => <ActionModal { ...props }
       actionType={ 'Add Comment' }
-      preMethod = { fetchRequest }
+      fetchData = { true }
```

Comment:
There is a convention If you want to pass `true` boolean value as a prop. You can should just use its name:
```
<Component ... fetchData />
```

Reply:
Thanks for pointing this out. I ended up not using this prop in the component, so I will remove it as a prop here, but I'll use the convention next time it's needed.